docs/designs/conceptual-localization.md

```diff
@@ -49,25 +49,6 @@ Below kinds of mappings are considered to be supported and there is a **strong c
     ```
     
     > NOTE: The loc org name can be different, it should be configurable
-    
-  - **RepositoryAndFolder**, localization files are stored in ONE **different repository** for **all locales** under different **locale folder**
```

Comment:
>  - **RepositoryAndFolder**, localization files are stored in ONE **different repository** for **all locales** under  [](start = 0, length = 117)

so we'd like to deprecate this functionality?

Reply:
this option is not supported by docfx, just forgot to remove it.